chain/chain/src/chain.rs

```diff
@@ -4880,7 +4880,13 @@ impl<'a> ChainUpdate<'a> {
         let approvals = header.approvals();
         self.runtime_adapter.verify_approvals_and_threshold_orphan(
             epoch_id,
-            self.doomslug_threshold_mode,
+            &|approvals, stakes| {
```

Comment:
I think this delta belonged to another PR.. hmm..

Reply:
Yeah, until https://github.com/near/nearcore/pull/7657 is actually merged, the diff would suck. The best advise I have is to review per commit, manually selecting the right one to start from